src/runtime_src/core/tools/common/tests/aie_pl_util/pl_controller_aie2.cpp

```diff
@@ -122,27 +121,6 @@ void plController_aie2::enqueue_loop_end() {
     m_opcodeBuffer.add(CMD_TYPE::LOOP_END);
 }
 
-void plController_aie2::enqueue_set_and_enqueue_dma_bd(const std::string& portName, int idx, int dma_bd_len, int id) {
-    auto buffers = get_buffers(portName);
-    if (buffers.size() == 0)
-        throw std::runtime_error("Cannot find port " + portName);
-    else if (static_cast<long unsigned int>(idx) > buffers.size() - 1)
-        throw std::runtime_error("port idx " + std::to_string(idx) + "is out of range");
-    auto buffer = buffers.at(idx);
-
-    uint32_t dma_bd_value = 0x83FC0000 + dma_bd_len - 1;
-    m_opcodeBuffer.add(CMD_TYPE::SET_DMA_BD);
-    m_opcodeBuffer.add(buffer.bd_num);
-    m_opcodeBuffer.add(dma_bd_value);
-    m_opcodeBuffer.add(id);
-
-    m_opcodeBuffer.add(CMD_TYPE::ENQUEUE_DMA_BD);
-    m_opcodeBuffer.add(buffer.bd_num);
-    m_opcodeBuffer.add(buffer.ch_num);
-    m_opcodeBuffer.add(buffer.s2mm);
-    m_opcodeBuffer.add(id);
-}
-
```

Comment:
removed the plController_aie2::enqueue_set_and_enqueue_dma_bd() since its not being used on aie2 pl test. Another reason it calls get_buffers() which uses reference to dma_lock_report.json file. On aie2 test there is no reference to this json file.